### ISSUE TITLE
Mozilla Firefox 32 bit crashes on a 64 bit Windows 10 machine

### ISSUE THREAD

**Issue by NZGene** [nvaccess/nvda#8731]
#### Have you tried any other versions of NVDA? Yes have tried stable, alpha and beta. Does not crash on alpha or stable!
[nvda.log](https://github.com/nvaccess/nvda/files/2365445/nvda.log)

**Comment by Brian1Gaff** [comment]
Are there any  firefox add ons on your version which crashes?

bglists@blueyonder.co.uk 
Sent via blueyonder. 
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
-----

**Comment by NZGene** [comment]
Hi Brian,

No there aren't - only the ones that came with Firefox. I disabled them and re-enabled them to see if that was the problem and it didn't appear to be. It seems to be the 2018.3 beta and RC version that only crash on the 32 bit version of Firefox 61 and Firefox 62 on my 64 bit computer. This is on both the installer version and the portable version. Yet, the beta and the RC versions work on the 64 bit version of Firefox 61 and Firefox 62 on my 64 bit machine (both portable and installer versions).

**Comment by lukaszgo1** [comment]
@NZGene Is it fixed for you with NVDA 2018.3.1? If so, this issue should be closed.

**Comment by NZGene** [comment]
yes fixed as of the release yesterday with 2018.3.1
